include/fiction/algorithms/simulation/sidb/is_ground_state.hpp

```diff
@@ -5,45 +5,66 @@
 #ifndef FICTION_IS_GROUND_STATE_HPP
 #define FICTION_IS_GROUND_STATE_HPP
 
-#include "fiction/algorithms/simulation/sidb/minimum_energy.hpp"
-#include "fiction/algorithms/simulation/sidb/quickexact.hpp"
-#include "fiction/algorithms/simulation/sidb/quicksim.hpp"
+#include "fiction/algorithms/simulation/sidb/determine_groundstate_from_simulation_results.hpp"
 #include "fiction/algorithms/simulation/sidb/sidb_simulation_result.hpp"
 #include "fiction/traits.hpp"
-#include "fiction/utils/math_utils.hpp"
 
-#include <cmath>
+#include <cstdint>
+#include <unordered_set>
 
 namespace fiction
 {
 
 /**
- * This function checks if the ground state is found by the *QuickSim* algorithm.
+ * This function checks if the ground state is found by the heuristic.
  *
  * @tparam Lyt Cell-level layout type.
  * @param heuristic_results All found physically valid charge distribution surfaces obtained by a heuristic algorithm.
- * @param exhaustive_results All valid charge distribution surfaces determined by ExGS.
+ * @param exact_results All valid charge distribution surfaces determined by ExGS.
  * @return Returns `true` if the relative difference between the lowest energies of the two sets is less than
  * \f$0.00001\f$, `false` otherwise.
```

Comment:
Why this particular number? Should we make it a parameter with this value as default?

Reply:
sorry, outdated